modules/commerce_bluesnap_currency/commerce_bluesnap_currency.info.yml

```diff
@@ -0,0 +1,8 @@
+name: Commerce Bluesnap Currency
+type: module
+description: 'Provides currency resolver which fetches currency conversion rates from Bluesnap.'
+package: Commerce (contrib)
+core: 8.x
+dependencies:
+  - commerce:commerce_bluesnap
```

Comment:
**CHANGE REQUEST**

The first part of the dependency is not the package, is the parent module:

```
- commerce_bluesnap:commerce_bluesnap
- commerce_currency_resolver
```

Reply:
Modified. Commerce bluesnap currency does not actually depend on commerce bluesnap module now. Hence removed it from dependency

Reply:
You still have `commerce_currency_resolver:commerce_currency_resolver`. If a module does not have submodules I don't think you need the prefix, just `commerce_currency_resolver` will do. Please verify though.

Reply:
@krystalcode 
The documentation in d.o says
`dependencies: a list of other modules your module depends on. Dependencies should be namespaced in the format {project}:{module}, where {project} is the project name as it appears in the Drupal.org URL (e.g. drupal.org/project/views) and {module} is the module's machine name.`
So for in here the project name and module machine name is  drupal.org/project, so it should read
`commerce_currency_resolver:commerce_currency_resolver` - correct?

Reply:
yeah you're right.